flowermd/tests/base/test_simulation.py

```diff
@@ -357,3 +357,10 @@ def test_gsd_logger(self, benzene_system):
 
         os.remove("trajectory.gsd")
         os.remove("sim_data.txt")
+
+    def test_flush(self, benzene_system):
+        sim = Simulation.from_system(benzene_system)
+        sim.run_NVT(kT=1.0, tau_kt=0.01, n_steps=500)
+        sim.flush_writers()
+        assert os.path.isfile("trajectory.gsd")
```

Comment:
I think the `trajectory.gsd` file is actually being created when the simulation starts, but it will be empty unless the writer is flushed. We could test that number of frames > 0 after flushing.

Reply:
Good point, I think changing `write_at_start` to `False` should fix this.